pkg/logs/message/message.go

```diff
@@ -20,7 +20,7 @@ import (
 var TruncatedFlag = []byte("...TRUNCATED...")
 
 // TruncatedTag is added to truncated log messages (if enabled).
-var TruncatedTag = "truncated:true"
+const TruncatedTag = "truncated:true"
```

Comment:
Since our backend doesn't require tag to have a value, and there are no other values of truncated to distinguish, would something like this make sense?

```suggestion
const TruncatedTag = "truncated"
```